SampleGallery/Samples/SDK 16299/LinearGradients/LinearGradients.xaml.cs

```diff
@@ -0,0 +1,257 @@
+
+            // Create the four visuals that will be used to hold the LG brush
+            _vis1 = _compositor.CreateSpriteVisual();
+            _vis2 = _compositor.CreateSpriteVisual();
+            _vis3 = _compositor.CreateSpriteVisual();
+            _vis4 = _compositor.CreateSpriteVisual();
+
+            // Create the purple to orange brush that will be used on all visuals
+            _brush1 = _compositor.CreateLinearGradientBrush();
+            b1GradientStop1 = _compositor.CreateColorGradientStop();
+            b1GradientStop1.Offset = 0;
+            b1GradientStop1.Color = Colors.DeepPink;
+            b1GradientStop2 = _compositor.CreateColorGradientStop();
+            b1GradientStop2.Offset = 1;
+            b1GradientStop2.Color = Colors.Honeydew;
+            _brush1.ColorStops.Add(b1GradientStop1);
+            _brush1.ColorStops.Add(b1GradientStop2);
+            
+
+            // Paint visuals with brushes and set their locations
+            _vis1.Brush = _brush1;
+            _vis1.Scale = new Vector3(0, 1, 0);
+            _vis1.Size = new Vector2((float)Rectangle1.ActualWidth, (float)Rectangle1.ActualHeight);
+
+
+            _vis2.Brush = _brush1;
+            _vis2.Scale = new Vector3(0, 1, 0);
+            _vis2.Size = new Vector2((float)Rectangle2.ActualWidth, (float)Rectangle2.ActualHeight);
+
+            
+            _vis3.Brush = _brush1;
+            _vis3.Scale = new Vector3(0, 1, 0);
+            _vis3.Size = new Vector2((float)Rectangle3.ActualWidth, (float)Rectangle3.ActualHeight);
+
+            _vis4.Brush = _brush1;
+            _vis4.Scale = new Vector3(0, 1, 0);
+            _vis4.Size = new Vector2((float)Rectangle4.ActualWidth, (float)Rectangle4.ActualHeight);
+
+            // Parent visuals to XAML rectangles
+            ElementCompositionPreview.SetElementChildVisual(Rectangle1, _vis1);
+            ElementCompositionPreview.SetElementChildVisual(Rectangle2, _vis2);
+            ElementCompositionPreview.SetElementChildVisual(Rectangle3, _vis3);
+            ElementCompositionPreview.SetElementChildVisual(Rectangle4, _vis4);
+
+
+            //create the scale & offset animation
+            Vector3KeyFrameAnimation offsetAnim_r1 = _compositor.CreateVector3KeyFrameAnimation();
+            offsetAnim_r1.InsertKeyFrame(1, new Vector3((float)Rectangle1.ActualWidth, (float)Rectangle1.ActualHeight, 0));
+            offsetAnim_r1.Duration = TimeSpan.FromSeconds(1);
+
+            _scaleAnim = _compositor.CreateVector3KeyFrameAnimation();
+            _scaleAnim.InsertKeyFrame(0, new Vector3(0, 1, 0));
+            _scaleAnim.InsertKeyFrame(.5f, new Vector3(1, 1, 0));
+            _scaleAnim.InsertKeyFrame(1, new Vector3(0, 1, 0));
+            _scaleAnim.Duration = TimeSpan.FromSeconds(2);
+
+
+            // animation of color stops
+            _colorAnimBrush1Stop1 = _compositor.CreateColorKeyFrameAnimation();
+            _colorAnimBrush1Stop1.InsertKeyFrame(.5f, Colors.Honeydew);
```

Comment:
Rather than hardcoding specific colors everywhere, could we put them in constants that then have more descriptive names describing their use?  Ex rather than "Colors.Honeydew/Colors.DeepPink" "c_lightColor/c_darkColor"